site/gatsby-site/src/components/checklists/CheckListForm.js

```diff
@@ -0,0 +1,437 @@
+      title: 'Goals',
+      id: 'tags_goals',
+      idValue: values['tags_goals'],
+      labelKey: abbreviatedTag,
+      include: (tagParts) =>
+        tagParts[0] == 'GMF' && ['Known AI Goal', 'Potential AI Goal'].includes(tagParts[1]),
+      ...{ tags, setFieldValue },
+      placeHolder: 'Goals for which to list associated risks',
+    }}
+  />
+);
+
+const MethodsTagInput = ({ values, tags, setFieldValue }) => (
+  <QueryTagInput
+    {...{
+      title: 'Methods',
+      id: 'tags_methods',
+      idValue: values['tags_methods'],
+      labelKey: abbreviatedTag,
+      include: (tagParts) =>
+        tagParts[0] == 'GMF' &&
+        ['Known AI Technology', 'Potential AI Technology'].includes(tagParts[1]),
+      ...{ tags, setFieldValue },
+      placeHolder: 'Methods for which to list associated risks',
+    }}
+  />
+);
+
+const OtherTagInput = ({ values, tags, setFieldValue }) => (
+  <QueryTagInput
+    {...{
+      title: 'Other',
+      id: 'tags_other',
+      idValue: values['tags_other'],
+      labelKey: (tag) => tag,
+      include: (tagParts) =>
+        tagParts[0] != 'GMF' ||
+        ![
+          'Known AI Goal',
+          'Potential AI Goal',
+          'Known AI Technology',
+          'Potential AI Technology',
+        ].includes(tagParts[1]),
+      ...{ tags, setFieldValue },
+      placeHolder: 'Other tags for which to list associated risks',
+    }}
+  />
+);
+
+const Header = classy('header', 'titleWrapper');
+
+const HeaderRow = classyDiv('w-full flex items-center flex-wrap justify-between gap-3');
+
+const HeaderControls = classyDiv(
+  'flex flex-wrap md:flex-nowrap shrink-0 gap-2 items-center max-w-full'
+);
+
+const SideBySide = classyDiv(
+  'flex flex-col md:flex-row gap-2 [&>*]:w-full [&>*]:md:w-1/2 [&>*]:h-full'
+);
+
+function SavingIndicator({ saveStatus }) {
+  const className = 'text-lg text-gray-500 inline-block mx-4';
+
+  if (!saveStatus) {
+    return <></>
+  }
+  if (saveStatus == 'saving') {
+    return (
+      <span {...{ className }}>
+        <Spinner />
+        <Trans>Saving...</Trans>
+      </span>
+    );
+  }
+  if (saveStatus == 'saved') {
+    return (
+      <span {...{ className }}>
+        <Trans>Saved</Trans>
+      </span>
+    );
+  }
+  if (saveStatus == 'error') {
+    return (
+      <span {...{ className }}>
+        <Spinner />
+        <Trans>Error saving...</Trans>
+      </span>
+    );
+  }
+}
+
+function Info({ children, className }) {
+  //  return <>{children}</>
+  const [hide, setHide] = useState(false);
+
+  if (hide) return <></>;
+  return (
+    <div
+      className={`${className} bg-amber-50 text-amber-900 border border-amber-300 p-4 rounded shadow-md my-4`}
+    >
+      <button
+        className="border-0 bg-none float-right text-xl pl-4 mt-0 pb-2 -mt-2"
+        onClick={() => setHide(true)}
+      >
+        ×
+      </button>
+      <strong>INFO</strong>: {children}
+    </div>
+  );
+}
+
+// TODO: Group known and potential in GMF Taxonomy
```

Comment:
Create and link issue?

Reply:
https://github.com/responsible-ai-collaborative/aiid/issues/2302